ale/base/base.py

```diff
@@ -194,27 +235,51 @@ def sensor_model_version(self):
 
     @abc.abstractproperty
     def focal2pixel_lines(self):
+        """
+        Returns
+        -------
+        : int
```

Comment:
This is a 3 element array of affine transformation coefficients. The coefficients are constant, x coefficient, then y coefficient.  For example, if you have a focal plane coordinate (x,y) and this is F, then the pixel line coordinate is F[0] + x * F[1] + y * F[2].